src/Common.props

```diff
@@ -54,7 +55,6 @@
 
   <PropertyGroup>
     <CodeAnalysisRuleSet>..\rules.ruleset</CodeAnalysisRuleSet>
-    <RunCodeAnalysis>true</RunCodeAnalysis>
```

Comment:
Turns out this wasn't doing anything - https://github.com/dotnet/roslyn-analyzers/issues/2652

Reply:
Ah ha! That's why I couldn't get this to work. I was just disabling this and it wasn't doing anything/

Reply:
That's great news, because this is the setting that had broken Visual Studio. With it removed we can build and test in VS again.